tensorflow/python/ops/array_ops.py

```diff
@@ -3962,7 +3962,7 @@ def reverse_sequence(input,
   >>> seq_lengths = [7, 2, 3, 5]
   >>> input = [[1, 2, 3, 4, 5, 0, 0, 0], [1, 2, 0, 0, 0, 0, 0, 0],
   ...          [1, 2, 3, 4, 0, 0, 0, 0], [1, 2, 3, 4, 5, 6, 7, 8]]
-  >>> output = reverse_sequence(input, seq_lens, seq_dim=1, batch_dim=0)
+  >>> output = reverse_sequence(input, seq_lengths, seq_dim=1, batch_dim=0)
```

Comment:
Could you please update the code to call `tf.reverse_sequence` instead of just `reverse_sequence`

Reply:
Also it seems like this should be `seq_axis` instead of `seq_dim`

Reply:
@yashk2810: Is there a reason these errors weren't caught in our doctects?

Reply:
So I investigated and seems like that doctest is not being run at all. 

The reason has to do with the lines below it

```
reverse_sequence.__doc__ = deprecation.rewrite_argument_docstring(
    deprecation.rewrite_argument_docstring(
        gen_array_ops.reverse_sequence.__doc__, "batch_dim", "batch_axis"),
    "seq_dim", "seq_axis")
```

`deprecation` replaces the docstring with the docstring of `gen_array_ops.reverse_sequence` which doesn't seem to have the `>>>` (carets).

Hence, it doesn;t get tested. We will need to fix that for it to start getting tested.

Reply:
Thanks @jaingaurav and @yashk2810. I've updated the args.